pkg/kubelet/apis/config/v1beta1/defaults_test.go

```diff
@@ -64,25 +64,25 @@ func TestSetDefaultsKubeletConfiguration(t *testing.T) {
 						CacheUnauthorizedTTL: metav1.Duration{Duration: 30 * time.Second},
 					},
 				},
-				RegistryPullQPS:                           utilpointer.Int32(5),
+				RegistryPullQPS:                           ptr.To[int32](5),
 				RegistryBurst:                             10,
-				EventRecordQPS:                            utilpointer.Int32(50),
+				EventRecordQPS:                            ptr.To[int32](50),
 				EventBurst:                                100,
-				EnableDebuggingHandlers:                   utilpointer.Bool(true),
-				HealthzPort:                               utilpointer.Int32(10248),
+				EnableDebuggingHandlers:                   ptr.To(true),
+				HealthzPort:                               ptr.To[int32](10248),
 				HealthzBindAddress:                        "127.0.0.1",
-				OOMScoreAdj:                               utilpointer.Int32(int32(qos.KubeletOOMScoreAdj)),
+				OOMScoreAdj:                               ptr.To[int32](int32(qos.KubeletOOMScoreAdj)),
```

Comment:
```suggestion
				OOMScoreAdj:                               ptr.To[int32](qos.KubeletOOMScoreAdj),
```

Reply:
Got it , thanks :+1:

Reply:
Actually, the parameter of `ptr.To[int32]` should be `int32`.  If modified, will report an error like `cannot use qos.KubeletOOMScoreAdj (constant -999 of type int) as int32 value in argument to ptr.To[int32]`, so it will be better to keep it as it is now.  

Thanks :)

Reply:
This happens because the `qos` constants are explicitly typed; untyped integers are coerced to whatever type is required. Removing the `int` specifiers at the start of `pkg/kubelet/qos/policy.go` would fix this. In any case, it’s better not to repeat the type, see my suggestion.

Reply:
I see. I didn't understand https://github.com/kubernetes/utils/pull/283 and @bart0sh 's comment well before. I will modify it according to your suggestion.

Thanks :+1: